watertap/flowsheets/activated_sludge/ASM1_flowsheet.py

```diff
@@ -344,3 +307,227 @@ def function(unit):
         time_point=0,
     )
     print(stream_table_dataframe_to_string(stream_table))
+    m.fs.report()
+
+    comps = [
+        "S_I",
+        "S_S",
+        "X_I",
+        "X_S",
+        "X_BH",
+        "X_BA",
+        "X_P",
+        "S_O",
+        "S_NO",
+        "S_NH",
+        "S_ND",
+        "X_ND",
+    ]
+
+    # composition values for treated effluent after METAB
+    comp_vals = [
+        0.086312003,
+        0.039661279,
+        0.11152474,
+        0.749114341,
+        1.40e-10,
+        1.40e-10,
+        1.40e-10,
+        1.40e-10,
+        1.40e-10,
+        0.205778397,
+        0.005252493,
+        0.058108176,
+    ]
+
+    m.fs.R3.KLa.fix(20)
+    m.fs.R4.KLa.fix(20)
+    m.fs.R5.KLa.fix(20)
+    new_comps = {comps[i]: comp_vals[i] for i in range(len(comps))}
+    solver = get_solver()
+    # solve_stat = {}
+    # clones = {}
+    # for i, (k,v) in enumerate(new_comps.items()):
+    #     clone_name = f"clone_change_{k}"
+    #     m_clone = m.clone()
+    #     old_val = pyo.value(m_clone.fs.feed.conc_mass_comp[0,k])
+    #     m_clone.fs.feed.conc_mass_comp[0,k].fix(v)
+    #     print(f"{k} was fixed.")
+    #     res = solver.solve(m_clone, tee=True)
+    #     if pyo.check_optimal_termination(res):
+    #         solve_stat[k]=1
+    #     else:
+    #         solve_stat[k]=0
+
+    #     clones[clone_name] = m_clone
+    #     m_clone.fs.feed.conc_mass_comp[0,k].fix(old_val)
+    #     del m_clone
```

Comment:
@Morgan88888888  This bit here is for troubleshooting which component concentration might be problematic in terms of model convergence; it checks the converge by fixing concentration values one at a time and independently. 

Uncomment this to (1) clone model, (2)  fix concentration to new value for that one component, (3) attempt to solve that cloned model and store optimal termination as 1 and failure as 0 in `solve_stat`, (4) store model clone in `clones` dict for optional diagnostics later, (5) delete clone and restart, cycling to the next component to probe

Just seeing now that the steps where I record original default value and refix the original default value are not necessary anymore since I moved to just cloning the model in each iteration.